include/actuator/ActuatorFunctorsSimple.h

```diff
@@ -101,11 +101,11 @@ struct ActSimpleComputeForce
     
   // Dual view polar tables and blade definitions
   ActScalarIntDv   polartable_size_;
-  const int        Npolartable;
+  const unsigned   Npolartable;
```

Comment:
@lawrenceccheung @psakievich While we are at it, can we also make this consistent with Nalu naming convention?

```
Npolartable -> nPolarTable_
Npts -> nPts_
twist_tableDv_ -> twistTableDv_
elem_areaDv_ -> elemAreaDv_
```

Reply:
yeah I can update those